src/pages/ServiceGraph/ServiceGraphPage.tsx

```diff
@@ -22,7 +21,7 @@ type ServiceGraphPageState = {
     namespaceAlert?: any;
     graphLoadAlert?: any;
   };
-  summaryData: any;
+  summaryData?: SummaryData | null;
```

Comment:
Just curious, what is the purpose of the  ' | null' ?

Reply:
@jshaughn it allows setting the value to `null` otherwise it is expected to be defined.

Reply:
The real question here for @israel-hdez , is do we ever actually set it to `null`? Or can it just be `undefined`?

Reply:
You can see in my changes that I replaced some:
```javascript
this.setState({
          // ...snip...
          summaryData: { summaryType: 'graph', summaryTarget: null },
          // ...snip...
        });
```

with:

```javascript
this.setState({
          // ...snip...
          summaryData: null,
          // ...snip...
        });
```

This was only to make simpler the conditional that decides if any side-panel should be present.

Reply:
The conditional is here: https://github.com/kiali/kiali-ui/pull/299/files#diff-bf5d5931e8a56754ac7cdfce71034a24R128